rasa/cli/test.py

```diff
@@ -101,6 +102,7 @@ def run_core_test(args: argparse.Namespace) -> None:
     else:
         test_core_models(args.model, stories, output)
 
+    print_info("Failed stories written to " + output + "/failed_test_stories.yml")
```

Comment:
Is there another way to get `failed_test_stories.yml` value? placing the name by hand is not a good idea because if the file changes will be needed to change this log by hand.